query07.sql

```diff
@@ -3,4 +3,11 @@
 */
 
 -- Enter your SQL query here
-select ...
+select count(*), '2019' as year 
+from public.indego_trips_2019_q2
+where extract(day from start_time) = extract(day from end_time) - 1
```

Comment:
It's a little more complicated than this, since you have to be aware of the month boundaries (e.g., April 30 -> May 1). You can see my solution for this filter [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/a2c00befb9eecbd166755a975708a98a55ad6906/query07.sql#L16-L25) (or a simpler version [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/a2c00befb9eecbd166755a975708a98a55ad6906/query07.sql#L40-L42)).